### PR TITLE
Fix logo bg, update logo

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Update KUH logo, fix dark mode backgrounds so it doesn't look like we gave the logo a new shape - that would violate the graphics profile.

Screenshot 2022-06-13 at 13 13 50

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
Nice! Would it be possible to add a white background also to the GMS logo, so it will look similar to the other ones?

**Comment by dnil** [comment]
The thought crossed my mind. I can check if they have a positive logo! That one is negative, so the text would not be visible on white. Also it might be a large white area in the end if we keep the separation and don't want to add borders around the GMS logo either that might look like it has a new shape.

**Comment by northwestwitch** [comment]
> The thought crossed my mind. I can check if they have a positive logo! That one is negative, so the text would not be visible on white. Also it might be a large white area in the end if we keep the separation and don't want to add borders around the GMS logo either that might look like it has a new shape.

Check how it looks like, otherwise this PR is already fine I think!

File: scout/server/blueprints/public/templates/public/index.html [hunk 1]
```diff
@@ -63,12 +63,12 @@ <h1 class="display-4">Scout</h1>
     {% endif %}
 
     <p class="pb-1 mb-0">Developed by:</p>
-    <div class="row justify-content-center align-items-center">
+    <div class="row justify-content-center align-items-center bg-white rounded-3">
```
Comment (northwestwitch) [inline scout/server/blueprints/public/templates/public/index.html:66]: 👍🏻

**Review by northwestwitch — APPROVED** [review]
Nice!

**Comment by dnil** [comment]
> > The thought crossed my mind. I can check if they have a positive logo! That one is negative, so the text would not be visible on white. Also it might be a large white area in the end if we keep the separation and don't want to add borders around the GMS logo either that might look like it has a new shape.
> 
> Check how it looks like, otherwise this PR is already fine I think!

If I switch to light mode - not that anyone would like to do that knowingly, mind you - but assuming that some might not have changed their settings yet, the GMS logo text is not visible on light mode. I'll try giving it a row of darker bg, or ask for a positive logo. They do have it in some presentation, but no material online that I found.

**Comment by dnil** [comment]
I've asked for a positive color GMS logo, and should probably ask for a negative KUH-one, but in the meantime this should comply with their profiles, if not being the height of elegance.

**Pull request merged**